routing/router.go

```diff
@@ -2293,3 +2293,265 @@ func generateBandwidthHints(sourceNode *channeldb.LightningNode,
 
 	return bandwidthHints, nil
 }
+
+var (
+	errDisconnectedSequence = errors.New("disconnected channel sequence")
+
+	errUnknownPolicy = errors.New("unknown channel policy")
+
+	errIncompatiblePolicies = errors.New("incompatible channel policies")
+)
+
+// BuildRoute returns a fully specified route based on a list of channels.
+func (r *ChannelRouter) BuildRoute(source route.Vertex, amt *lnwire.MilliSatoshi,
+	addresses []interface{}, finalCltvDelta int32, useMinAmt bool) (
```

Comment:
Nit: include the expected types of `addresses` in the godoc.

Reply:
Addresses removed, but go doc expanded on new params.